### DISCUSSION TITLE
Remove 'X is shared across all languages' popup warning for culture-invariant items

### DISCUSSION THREAD

**Discussion by danpikey** [umbraco/Umbraco-CMS#14411] [Features and ideas]
My client is finding this feature v annoying. Is there any way this can be changed so they don't have to click confirm before editing each content section? Once per document then just some non-blocking highlighting for the fields concerned?

![image](https://github.com/umbraco/Umbraco-CMS/assets/23554658/d833dd8e-be07-4d56-8eda-01074984b2e4)

**Comment by nul800sebastiaan** [comment]
Hey @danpikey - please add your thoughts to the existing discussion here: https://github.com/umbraco/Umbraco-CMS/discussions/14048
